pkg/lib/serverTime.js

```diff
@@ -321,13 +309,89 @@
                     console.warn("failed to load time servers", error);
                     return result;
                 });
-    };
+    }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18298-20230207-124146-95ee9c44-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)